CHANGES.md

```diff
@@ -1,3 +1,12 @@
+## v0.2.1 (2017-06-15)
+
+* Use [@@noalloc] rather than "noalloc"
```

Comment:
Doesnt this require ocaml-version>=4.03.0?

It will silently pass on 4.02.3 as the attribute didnt exist there yet